README.md

```diff
@@ -1,5 +1,5 @@
 # InstaTonne
-[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature. Our aim is to provide users with more autonomy and choice while connecting them to various independent social networks without being limited to a single platform like Facebook, Instagram or Twitter.
+[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature. 

```

Comment:
Sorry, I should have clarified; the references to other platforms are fine, I mostly wanted the grammatical error fixed

Reply:
Did I resolve the grammar error? Cause I really don't know what grammar error you were talking about. Let me know if there's still something needing fixing